app/services/email_sender_service/email_sender.rb

```diff
@@ -0,0 +1,45 @@
+module EmailSenderService
```

Comment:
I'd consider switching this to be a class of EmailSenderService rather than nesting a class of the same name in the module.

Reply:
do away with the namespacing? 😱 

perhaps we should have a general chat about the naming here as well as the underlying Notify and Pseudo classes sitting in the EmailSenderService namespace.

I was wondering if DeliveryService would be a better name at the module/namespace level and under that have EmailService as our orchestrator of Notify/Pseudo/any other provider?

Reply:
You'd still have namespacing just would be:

```
class EmailSenderService
   class Notify; end
   class Pseudo; end
end
```

rather than 

```
module EmailSenderService
   class EmailSender; end
   class Notify; end
   class Pseudo; end
end
```

Reply:
👍 so I've re-implemented this in https://github.com/alphagov/email-alert-api/pull/215
with the suggested structure